pkg/querier/batch/stream.go

```diff
@@ -122,10 +124,18 @@ func mergeStreams(left, right batchStream, result batchStream, size int) batchSt
 			right.next()
 		} else {
 			if (rt == chunkenc.ValHistogram || rt == chunkenc.ValFloatHistogram) && lt == chunkenc.ValFloat {
-				// Prefer histograms over floats. Take left side if both have histograms.
+				// Prefer histograms than floats. Take left side if both have histograms.
```

Comment:
nit, I think it was correct
```suggestion
				// Prefer histograms over floats. Take left side if both have histograms.
```